SPECS-EXTENDED/neon/neon.spec

```diff
@@ -1,19 +1,38 @@
 Vendor:         Microsoft Corporation
 Distribution:   Azure Linux
+
+
 %bcond_without tests
+%if 0%{?fedora}
+%bcond_without pkcs11
+%else
+%bcond_with pkcs11
+%endif
+%if 0%{?fedora}
+%bcond_without libproxy
+%else
+%bcond_with libproxy
+%endif
+
+# Disable automatic .la file removal
+%global __brp_remove_la_files %nil
 
 Summary: An HTTP and WebDAV client library
 Name: neon
-Version: 0.31.2
-Release: 2%{?dist}
-License: LGPLv2+
+Version: 0.33.0
+Release: 1%{?dist}
+License: LGPL-2.0-or-later
 URL: https://notroj.github.io/neon/
 Source0: https://notroj.github.io/neon/neon-%{version}.tar.gz
-Patch0: neon-0.27.0-multilib.patch
-BuildRequires:  gcc-c++
-BuildRequires: expat-devel, openssl-devel, zlib-devel, krb5-devel, libproxy-devel
-BuildRequires: pkgconfig, pakchois-devel
-Requires: ca-certificates
+Patch0: neon-0.32.2-multilib.patch
+BuildRequires: expat-devel, openssl-devel, zlib-devel, krb5-devel
```

Comment:
Can you please have a BuildRequires for each of these packages in a separate line.